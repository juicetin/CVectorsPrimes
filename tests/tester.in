4 8
200 2 0
vector= random 2354
vector= descending 0


avx instructions

3, 6, 9, 12
15, 18, 21, 24
27, 30, 33, 36
39, 42, 45, 48
51, 54, 57, 60