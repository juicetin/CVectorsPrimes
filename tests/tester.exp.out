median of vector~0 = 16445
